bundles/cis-k8s-v1.5.1/5.2.4_psp-host-network-ports.yaml

```diff
@@ -11,24 +11,38 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-
 apiVersion: constraints.gatekeeper.sh/v1beta1
 kind: K8sPSPHostNetworkingPorts
 metadata:
-  name: psp-host-network-ports
+  name: cis-k8s-v1.5.1-psp-host-network-ports
+  labels:
+    policycontroller.gke.io/bundleName: cis-k8s-v1.5.1
   annotations:
     # This constraint is not certified by CIS.
-    description: "Restricts containers from running with the `hostNetwork` flag set to `true`."
-    bundles.validator.forsetisecurity.org/cis-k8s-v1.5.1: 5.2.4
+    policycontroller.gke.io/constraintData: |
+      "{
+        bundleName: 'cis-k8s-v1.5.1',
+        bundleDisplayName: 'CIS Kubernetes Benchmark 1.5.1',
+        bundleLink: 'https://cloud.google.com/anthos-config-management/docs/how-to/using-cis-k8s-benchmark',
+        bundleVersion: '202208.1',
+        bundleDescription: 'Use the CIS Kubernetes Benchmark 1.5.1 policy bundle with Anthos Policy Controller to evaluate the compliance of your in cluster resources against the CIS Kubernetes Benchmark, which is a set of recommendations for configuring Kubernetes to support a robust security posture.',
+        controlNumbers: '[5.2.4]',
+        severity: 'UNASSIGNED',
+        description: 'Restricts containers from running with the `hostNetwork` flag set to `true`.',
+        remediation: 'Containers are restricted from running with `hostNetwork` set to `true`, please remove the `hostNetwork: true` from your containers.',
+        minimumTemplateLibraryVersion: '1.11.1'
+      }"
 spec:
   enforcementAction: dryrun
   match:
     kinds:
     - apiGroups:
-      - ''
+      - ""
       kinds:
       - Pod
+    excludedNamespaces:
+    - kube-system
   parameters:
-    hostNetwork: true
-    #min: 80
-    #max: 9000
+    hostNetwork: false
```

Comment:
@apeabody @kaariger We're using these constraints at our project. Can you comment on why this was changed from `true` to `false`?